spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/mapping/DatastoreMappingContext.java

```diff
@@ -55,6 +64,48 @@ public void setApplicationContext(ApplicationContext applicationContext)
 		this.applicationContext = applicationContext;
 	}
 
+	/**
+	 * Registers in the DatastoreMappingContext that two classes are discriminated from the
+	 * same Datastore Kind.
+	 * @param classA one class.
+	 * @param classB another class.
+	 */
+	public static void addDiscriminationClassConnection(Class classA, Class classB) {
```

Comment:
we probably want to test this method

Reply:
This one is tested in the method `testDiscriminationMetadata` in `DatastorePersistentEntityImplTests`. In that method I test that the sets returned as what we expect and they can only be that way because this method is being used.

Reply:
I see, makes sense.